mllib/src/main/scala/org/apache/spark/ml/recommendation/ALS.scala

```diff
@@ -82,12 +82,22 @@ private[recommendation] trait ALSModelParams extends Params with HasPredictionCo
    * Attempts to safely cast a user/item id to an Int. Throws an exception if the value is
    * out of integer range.
```

Comment:
Should probably add to comment "... out of integer range or contains a fractional part"